Documentation/Tracing.md

```diff
@@ -3,7 +3,7 @@
 ## Advantages over KdPrint:
 
 * It can be enabled, disabled and filtered during runtime with minimal overhead by logging in real-time binary messages
-* Traces are are automatically included in the crash dumps
+* Traces are automatically included in the crash dumps
```

Comment:
Duplicate word handling is a fairly new feature and I'm still gathering experience w/ it, but I'm fairly certain this change is correct.